app/shared-components/SteppedNavigation/__tests__/__snapshots__/SteppedNavigation.test.jsx.snap

```diff
@@ -350,20 +320,16 @@ exports[`app/shared-components/SteppedNavigation Snapshot tests with react-test-
       >
         <a
           aria-current={null}
-          className={undefined}
           href="/1"
           onClick={[Function]}
-          style={undefined}
         />
         <a
           className="active"
         />
         <a
           aria-current={null}
-          className={undefined}
```

Comment:
I don't know when all of these undefined props were added to the snapshots, but it's definitely good that they were removed with the newly generated ones.

Reply:
@trickpattyFH20 — @IAmThePan said that it may be best to revert back to the last snapshots with those undefined props in them. We did have snapshots with undefined props in the past that passed the tests, but I'm not sure if there's a setting that needs to be changed in a Jest config file or something like that.